### PR TITLE
Fixed SimpleAudioEngine::playEffect() lagged on Android 5.0.x

### PR DESCRIPTION
**Pull request by WenhaiLin**
Wrap AudioEngine to SimpleAudioEngine interfaces for Android 5.0.x.

### CODE REVIEW

File: cocos/audio/android/jni/cddandroidAndroidJavaEngine.cpp [hunk 1]
```diff
@@ -36,282 +39,463 @@ THE SOFTWARE.
...
+    : _implementBaseOnAudioEngine(false)
+    , _effectVolume(1.f)
+{
+    char sdk_ver_str[PROP_VALUE_MAX] = "0";
+    auto len = __system_property_get("ro.build.version.sdk", sdk_ver_str);
+    if (len > 0)
+    {
+        auto sdk_ver = atoi(sdk_ver_str);
+        __android_log_print(ANDROID_LOG_DEBUG, "cocos2d", "android build version:%d", sdk_ver);
+        if (sdk_ver == 21)
```
Comment (jimrange) [inline cocos/audio/android/jni/cddandroidAndroidJavaEngine.cpp:64]: @zilongshanren  Why is this sdk_ver == 21? Why not  sdk_ver  >= 21?
Does Android API 22 Android 5.1 LOLLIPOP_MR1 fix the [play effect lagged] issue?

**Comment by jimrange** [comment]
Does this bug exist if you target API 21?
Or does it exist if any target version is run on a device that has API 21 installed?
i.e. I target API-13 and run on a device that has API 21 installed. Will the lag occur?

**Comment by WenhaiLin** [comment]
@jimrange This issue irrelevant to target API.

**Comment by xiaofengxyz** [comment]
I have replaced the folder “cocos/audio” in cocos2dx 3.2 with  v3.3, but it does not work. What should I do to fix it on v3.2. Please help me.

**Comment by jminht** [comment]
@xiaofengxyz 
did you fix it? could u help me?
i'm using v3.1.1 and i found that the audio part is same with v3.2.
@WenhaiLin 
how can i fix it?

**Pull request merged**